packages/govuk-frontend/src/govuk/components/task-list/template.test.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @jest-environment jsdom
```

Comment:
Can get rid of all these lines now 😊

```js
/**
 * @jest-environment jsdom
 */
/* eslint-env jest */
```

We automatically match filenames for Jest and ESLint environments

Reply:
Done, thanks.